e2e/generate-test.mjs

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-console, import/no-extraneous-dependencies */
+import prompts from 'prompts'
+import fs from 'node:fs/promises'
+import dedent from 'dedent'
+
+(async () => {
```

Comment:
I thought esm supports root level await?

Reply:
I'm not sure, I got this from the `prompts` example from their readme. Perhaps it's not required.

Reply:
Using top level await is preferable, it will exit with a non-zero exit code if there is an unfinished await (e.g., `await new Promise(() => {})`) and report the exception if the awaited promise is rejected.

Reply:
`prompts` claims to support all Node.js 6+, TLA is available on Node.js 14+, if I had to guess, that's why they are not using it on their examples. Uppy should though.